tensorflow/python/kernel_tests/random/random_ops_test.py

```diff
@@ -413,6 +414,14 @@ def testSingleSessionGraphSeedNotConstant(self):
             use_gpu=use_gpu,
             graph_seed=965)
 
+  def testUniformWithInvalidMaxMindShape(self):
+    # Test case for GitHub issue 34363.
+    with self.assertRaisesRegexp(
+        errors.InvalidArgumentError,
+        "must be no greater than rank of output shape"):
+      array = array_ops.zeros(shape=(1,))
+      random_ops.random_uniform(shape=(), minval=array)
+
```

Comment:
`tensorflow/python/framework/ops.py", line 1820, in _create_c_op
    c_op = pywrap_tf_session.TF_FinishOperation(op_desc)
google3.third_party.tensorflow.python.framework.errors_impl.InvalidArgumentError: Shape must be at most rank 0 but is rank 1 for '{{node random_uniform/BroadcastTo_1}} = BroadcastTo[T=DT_FLOAT, Tidx=DT_INT32](zeros, random_uniform/shape)' with input shapes: [1], [0].

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<embedded stdlib>/unittest/case.py", line 59, in testPartExecutor
    yield
  File "<embedded stdlib>/unittest/case.py", line 605, in run
    testMethod()
  File "tensorflow/python/framework/test_util.py", line 1785, in decorated
    return func(self, *args, **kwargs)
  File "/tensorflow/python/kernel_tests/random/random_ops_test.py", line 425, in testUniformWithInvalidMaxMindShape
    random_ops.random_uniform(shape=(), minval=array)
  File "//tensorflow/python/util/dispatch.py", line 201, in wrapper
    return target(*args, **kwargs)
  File "tensorflow/python/ops/random_ops.py", line 312, in random_uniform
    minval = array_ops.broadcast_to(minval, shape)
  File "/tensorflow/python/ops/gen_array_ops.py", line 854, in broadcast_to
    "BroadcastTo", input=input, shape=shape, name=name)
  File "/tensorflow/python/framework/op_def_library.py", line 744, in _apply_op_helper
    attrs=attr_protos, op_def=op_def)
  File "//tensorflow/python/framework/ops.py", line 3493, in _create_op_internal
    op_def=op_def)
  File "/tensorflow/python/framework/ops.py", line 1983, in __init__
    control_input_ops, op_def)
  File "//tensorflow/python/framework/ops.py", line 1823, in _create_c_op
    raise ValueError(str(e))
ValueError: Shape must be at most rank 0 but is rank 1 for '{{node random_uniform/BroadcastTo_1}} = BroadcastTo[T=DT_FLOAT, Tidx=DT_INT32](zeros, random_uniform/shape)' with input shapes: [1], [0].`